controllers/nodeagent_test.go

```diff
@@ -385,6 +390,23 @@ func createTestBuiltNodeAgentDaemonSet(options TestBuiltNodeAgentDaemonSetOption
 								EmptyDir: &corev1.EmptyDirVolumeSource{},
 							},
 						},
+						{
+							Name: "bound-sa-token",
+							VolumeSource: corev1.VolumeSource{
+								Projected: &corev1.ProjectedVolumeSource{
+									Sources: []corev1.VolumeProjection{
+										{
+											ServiceAccountToken: &corev1.ServiceAccountTokenProjection{
+												Audience:          "openshift",
+												ExpirationSeconds: ptr.To(int64(3600)),
+												Path:              "token",
+											},
+										},
+									},
+									DefaultMode: ptr.To(int32(420)),
```

Comment:
is not easier to read if we use octal (`0644`) instead of decimal?

Reply:
No opinions on this front. I think this matches CSV integer. Can change

Reply:
nvm that was just showing up in unit tests.. will check again.

Reply:
This is the diff that showed up when this line is commented out.
```
            - 								DefaultMode: nil,
            + 								DefaultMode: &420,
```

I wasn't aware of octal.. but reusing the common.DefaultProjectedPermission you merged earlier worked.

But no, to me, it isn't easier to read, because I wasn't trying to read it.

Reply:
updated to using octal var from common.